kernel/util/mlog-base/util/Logger.hh

```diff
@@ -99,31 +99,31 @@ void Logger<Filter>::write(ARGS&&... args) const {
 }
 
 static Logger<FilterAny> testLog("Test");
```

Comment:
This variable gets a separate instance in every compilation object file. Thanks to static linkage, the linker won't complain. However, I would prefer `extern Logger<FilterAny> testLog;` here and the actual instance `Logger<FilterAny> testLog("Test");` in Logger.cc.